### PR TITLE
Table columns move

### PR DESCRIPTION
**Pull request by zvonimirfras**
depends on #2606

### CODE REVIEW

**Review by Donisius — APPROVED** [review]

**Review by davidicus — APPROVED** [review]

**Review by davidicus — APPROVED** [review]

**Review by davidicus — CHANGES_REQUESTED** [review]
blocking merge until we can merge from master

**Review by davidicus — APPROVED** [review]

**Pull request merged**
